tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/.runsettings

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+    <!-- Timeout in ms, 20 minutes -->
```

Comment:
I am trying to figure out if the MySQL tests are failing because of the time it takes only. There aren't any failures in the log but it stops at 15min, which was the timeout. Locally they all run in 5s. Two tests (with different inline data) are taking [1 m 25 s]: HealthCheckReportsExpectedStatus and LoggerFactoryIsUsedByRegisteredClient. They account for all the time on Helix. They take 100ms locally. And this should not be about downloading the image because these delays appear at different wall clock times in the log.

https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-aspire-refs-pull-5932-merge-b3281ba7f783453882/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests-net9.0/1/console.0d22ff48.log?helixlogtype=result

Reply:
Note that this run didn't take my change into account : `-- RunConfiguration.TestSessionTimeout=900000`, I expected 1200000 here.

Reply:
Finally, this is not specific to this branch, I took the most recent passing build and this test set took 17m12.

```
[xUnit.net 00:17:12.28]   Finished:    Aspire.Pomelo.EntityFrameworkCore.MySql.Tests&#xD;
```

Reply:
It's random:

![image](https://github.com/user-attachments/assets/8fb58269-b8e6-41a9-9f0c-33fff64233d1)

@radical can you see what I need to add on top of the .runsettings to increase this test timeout? Did something change in this PR related to it, since another successful run on another PR took 17m which is more than the default timeout I am seeing in this PR.

Reply:
The timeout for the helix job has to be set in the helix targets (till we get https://github.com/dotnet/aspire/pull/5505 merged).